README.md

```diff
@@ -1,13 +1,18 @@
 # Mongo API Project
+Build a Mongo database
 
-Replace this readme with your own information about your project. 
+## Learnings
+- Create routes
+- Create mongoose models
+- Deploy Node projects
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## TECH
+- Heruko
+- Express
+- Mongo DB
+- Mongoose
+- Postman
+- Javascript
 
```

Comment:
Your documentation does not show how to use your API. 
Adding the list of endpoints here would be a great enchantment :)

Reply:
updated :)